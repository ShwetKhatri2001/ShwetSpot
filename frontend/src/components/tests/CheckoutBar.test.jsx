import React from 'react';
import renderer from 'react-test-renderer';

import CheckoutBar from '../CheckoutBar';

it('should render correctly', () => {
  const tree = renderer.create(<CheckoutBar />).toJSON();
  expect(tree).toMatchSnapshot();
});
