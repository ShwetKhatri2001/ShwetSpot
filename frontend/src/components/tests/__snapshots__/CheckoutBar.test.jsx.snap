// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="justify-content-center mb-4 nav"
  onKeyDown={[Function]}
  style={
    Object {
      "backgroundColor": "#f8f8f8",
    }
  }
>
  <div
    className="nav-item"
  >
    <a
      aria-disabled={true}
      className="nav-link disabled"
      data-rb-event-key={null}
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      tabIndex={-1}
    >
      Sign In
    </a>
  </div>
  <div
    className="nav-item"
  >
    <a
      aria-disabled={true}
      className="nav-link disabled"
      data-rb-event-key={null}
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      tabIndex={-1}
    >
      Shipping
    </a>
  </div>
  <div
    className="nav-item"
  >
    <a
      aria-disabled={true}
      className="nav-link disabled"
      data-rb-event-key={null}
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      tabIndex={-1}
    >
      Payment
    </a>
  </div>
  <div
    className="nav-item"
  >
    <a
      aria-disabled={true}
      className="nav-link disabled"
      data-rb-event-key={null}
      href="#"
      onClick={[Function]}
      onKeyDown={[Function]}
      role="button"
      tabIndex={-1}
    >
      Place Order
    </a>
  </div>
</div>
`;
